### DISCUSSION TITLE
Something similar to VS Code Live Share

### DISCUSSION THREAD

**Discussion by bonsairobo** [helix-editor/helix#8568] [Ideas]
That would be nicer than having to use a shared tmux session over SSH. Are there any plans for this? I haven't seen anything mentioned in other discussions or issues.

**Comment by kirawi** [comment]
https://github.com/helix-editor/helix/issues/312

**Reply by bonsairobo** [reply]
AFAICT kakoune's client-server model does not quite offer the same experience as VS Code Live Share. 

https://discuss.kakoune.com/t/networked-collaboration/1286/2

In particular, several things are not properly isolated between users, like clipboard and file permissions.

**Reply by kirawi** [reply]
It seems like the intention is to build pair programming on top of the client-server model. You can also see https://github.com/helix-editor/helix/issues/312#issuecomment-899849977

**Reply by bonsairobo** [reply]
Thanks for linking that comment. I generally agree that having a solution which is editor-agnostic would be nice, but I suspect that would also be significantly more work and probably make sacrifices in what kinds of features are supported.

Anyway, I'm glad to see pair programming is a use case that's being considered. It's just not immediately obvious that #312 is the issue where such discussion should occur.

**Reply by kirawi** [reply]
You can create a separate issue to track the feature

**Reply by bonsairobo** [reply]
Will do.
